### DISCUSSION TITLE
子应用中有个谷歌第三方登录的按钮，为什么加载谷歌第三方登录按钮出现跨域呢？

### DISCUSSION THREAD

**Discussion by geekskai** [umijs/qiankun#1600] [Q&A]
下面是使用qiankun [搭建的微应用链接](https://dev-py.notta.io/home)

1. 打开控制台，刷新页面，出现报错
`Access to fetch at 'https://apis.google.com/_/scs/apps-static/_/js/k=oz.gapi.zh_CN.k5FXU-cZzhw.O/m=auth2/exm=client/rt=j/sv=1/d=1/ed=1/am=AQ/rs=AGLTcCNtV7Deo6ZjrCUSqzJBekYBlusVgA/cb=gapi.loaded_1?le=ili,ipu' from origin 'https://dev-py.notta.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

最后发现 谷歌登录的代码都被注释掉了
![image](https://user-images.githubusercontent.com/37830362/126785042-ae0e5508-9fbb-452a-8168-8ee82aabf47b.png)

**Comment by kuitos** [comment]
https://qiankun.umijs.org/zh/faq#%E5%BE%AE%E5%BA%94%E7%94%A8%E9%9D%99%E6%80%81%E8%B5%84%E6%BA%90%E4%B8%80%E5%AE%9A%E8%A6%81%E6%94%AF%E6%8C%81%E8%B7%A8%E5%9F%9F%E5%90%97%EF%BC%9F
